rmw_connext_cpp/src/rmw_take.cpp

```diff
@@ -215,38 +214,37 @@ _take_sequence(
   rmw_subscription_allocation_t * allocation)
 {
   (void) allocation;
-  if (!subscription) {
-    RMW_SET_ERROR_MSG("subscription handle is null");
-    return RMW_RET_INVALID_ARGUMENT;
-  }
+
+  RMW_CHECK_ARGUMENT_FOR_NULL(
+    subscription, RMW_RET_INVALID_ARGUMENT);
```

Comment:
Why this one doesn't return a msg and the one for `rmw_take` does? Same questions for the rest of the checks in `take_seqeuence`

Reply:
Great question. Actually, I forgot to get rid of the messages on the first ones. I got rid of all messages as @hidmic suggested, they were kind of random depending on each implementation, and pretty inaccurate, the default `RMW_CHECK_ARGUMENT_FOR_NULL` seems more appropriate.